File: training/java/src/main/java/de/sots/cellarsandcentaurs/domain/model/Centaur.java
package de.sots.cellarsandcentaurs.domain.model;

public class Centaur extends Creature {
    private static final int HOOVES_DAMAGE = 6;

    public Centaur(CreatureId id) {
        super(id, CreatureType.MONSTROSITY);
    }

    public int hoovesDamage() {
        return HOOVES_DAMAGE;
    }
}

File: training/kotlin/src/main/kotlin/de/sots/cellarsandcentaurs/domain/model/Centaur.kt
package de.sots.cellarsandcentaurs.domain.model

@Roams
class Centaur(id: CreatureId) : Creature(id, CreatureType.MONSTROSITY) {
    val gallopSpeed: Speed
        get() = speeds[SpeedType.WALKING] ?: Speed(50)
}
